react-native/react/engine/transport.desktop.js

```diff
@@ -14,17 +25,12 @@ export default class DesktopTransport extends BaseTransport {
       console.error('No keybased socket file found!')
     }
 
-    let hooks = null
-    if (connectCallback) {
-      hooks = {connected: connectCallback}
-    }
+    this.net_opts.path = sockfile
```

Comment:
don't you mean it kinda _socks_... 

![](http://www.reactiongifs.us/wp-content/uploads/2014/01/ba_dum_tss_pirates_band_of_misfits.gif)
I'll see myself out.